HiP-ThumbnailService/Utility/DirConfig.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.ThumbnailService.Utility
+{
+    public class DirConfig
+    {
+        /// <summary>
+        /// Path to thumbnail images
+        /// </summary>
+        public string Path { get; set; }
+
+        /// <summary>
+        /// Formats that are supported for file upload
+        /// </summary>
+        public List<string> SupportedFormats { get; set; }
```

Comment:
Unused configuration property. And I think `DirConfig` and `SizeConfig` could be combined in one class to keep things simple.